cfme/tests/test_appliance.py

```diff
@@ -158,6 +158,28 @@ def test_certificates_present(appliance, soft_assert):
 
 
 @pytest.mark.ignore_stream("upstream")
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.8', reason='Only valid for >5.8')
+def test_html5_ssl_files_present(appliance, soft_assert):
+    """Test if the certificate and key necessary for HTML 5 Console Support
+       is present.  These should have been generated by the
+       IPAppliance object.   Note, these files are installed by
+       the cfme RPM, so we use rpm verify to make sure they do not verify
+       and hence were replaced.
+    """
+    cert = conf.cfme_data['vm_console']['cert']
+    cert_file = os.path.join(cert.install_dir, 'server.cer')
+    key_file = os.path.join(cert.install_dir, 'server.cer.key')
+    ssl_files = [cert_file, key_file]
+
+    for ssl_file in ssl_files:
```

Comment:
**Optional**
Kind of preference here, but consider just using `for ssl_file in [cert_file, key_file]` since you don't re-use ssl_files anywhere else.